test/common/schemaform/reducers/index.unit.spec.js

```diff
@@ -50,10 +50,10 @@ describe('schemaform createSchemaFormReducer', () => {
     const reducer = createSchemaFormReducer(formConfig);
     const state = reducer(undefined, {});
 
-    expect(state.submission).to.be.defined;
+    expect(state.submission).not.to.be.undefined;
     expect(state.data.privacyAgreementAccepted).to.be.false;
     expect(state.data.field).to.eql(formConfig.chapters.test.pages.page1.initialData.field);
-    expect(state.page2).to.be.defined;
+    expect(state.page2).to.be.undefined;
```

Comment:
I don't think `defined` was a valid option before and Chai didn't fail because of it, which it does now. So, I think this exposed a test that was never properly working.